editoast/src/views/projection.rs

```diff
@@ -462,13 +427,13 @@ async fn retrieve_cached_projection(
     info!(
         nb_hit = hit_cache.len(),
         nb_miss = miss_cache.len(),
-        "Hit cache",
+        "Hit cache"
     );
 
     Ok((miss_cache, hit_cache))
 }
 
-async fn extract_train_details(
+pub async fn extract_train_details(
```

Comment:
That's unfortunate... but given how things are architectured nowadays, I don't have a better solution than this. The whole simulation stack will need to be reworked soon anyway to handle errors properly.

```suggestion
pub(in crate::views::timetable) async fn extract_train_details(
```

Reply:
we have to talk about it, I m really not sure to understand your suggestion